packages/oui-back-button/README.md

```diff
@@ -0,0 +1,42 @@
+# Back button
+
+<component-status cx-design="complete" ux="prototype"></component-status>
+
+## Usage
+
+```html
+<oui-back-button>
+</oui-back-button>
+```
+
+## Examples
+
+### Default
+
+```html:preview
+<oui-back-button>
```

Comment:
The back button (on CX specs) also includes the text at the right, do we want it ?

Reply:
not sure about this one, do we really want to restrict the component to be used with a h2? (similar to : do we add labels to input components)
if that's the case, it won't be a oui-back-button anymore but maybe something like oui-page-title (please give me great ideas ;) )

Reply:
Saw together, we will keep it simple.